packages/filebrowser/style/base.css

```diff
@@ -215,13 +215,36 @@
 .jp-DirListing-item {
   display: flex;
   flex-direction: row;
-  padding: 4px 12px;
+  align-items: center;
+  padding-right: 12px;
```

Comment:
Is there a right-to-left mode for JupyterLab that I should worry about?

Reply:
Right now, I believe the answer is no. Long term this is something we'd like to support (and have had [RTL support requested via issues](https://github.com/jupyterlab/jupyterlab/issues/1163)). (And [further discussion on this issue](https://github.com/jupyterlab/jupyterlab/issues/3046).)

Reply:
Thanks for that context!